src/sass/common/_Image.scss

```diff
@@ -0,0 +1,5 @@
+.Image {
+    &.error {
+        filter: blur(2px);
```

Comment:
will need to talk to UX about if they want to visually display failed images, this is temporary till I chat with em